### PR TITLE
Fixed bug: CheckboxTreeNode label overflows, and node icon disappears

### PR DESCRIPTION
**Pull request by geleems**
This is fix for the bug: https://github.com/Microsoft/azuredatastudio/issues/5021
The string value of `DIV` element with  `model-view-tree-node-item-label` class overflows if its container size is smaller than the string length, and this makes the tree node icon disappears.
This PR fixes the issue.

After fix: 
image

### CODE REVIEW

File: src/sql/parts/modelComponents/tree/treeComponent.css [hunk 1]
```diff
@@ -2,7 +2,12 @@
...
-.tree-component-node-tile .model-view-tree-node-item-icon{
-	width: 15px;
+.tree-component-node-tile .model-view-tree-node-item-icon {
+	width: 17px;
 	height: 15px;
+	flex-shrink: 0;
 }
+
+.tree-component-node-tile .model-view-tree-node-item-label {
+	overflow: hidden;
```
Comment (alanrenmsft) [inline src/sql/parts/modelComponents/tree/treeComponent.css]: try text-overflow:ellipsis

File: src/sql/parts/modelComponents/tree/treeComponent.css [hunk 2]
```diff
@@ -2,7 +2,12 @@
 	display: flex;
 }
 
-.tree-component-node-tile .model-view-tree-node-item-icon{
-	width: 15px;
+.tree-component-node-tile .model-view-tree-node-item-icon {
+	width: 17px;
 	height: 15px;
```
Comment (alanrenmsft) [inline src/sql/parts/modelComponents/tree/treeComponent.css]: is the image you are using svg? I think SVG will adjust it self.

Comment (geleems) [reply src/sql/parts/modelComponents/tree/treeComponent.css]: I found icon image set by 16px x 16px slightly miss aligned.
17px x 17px makes perfect alignment.

image

Comment (geleems) [reply src/sql/parts/modelComponents/tree/treeComponent.css]: `.model-view-tree-node-item-icon` class currently used only in Virtualize Data Extension.

**Pull request merged**